src/core/edit/edit.model.js

```diff
@@ -10,5 +10,11 @@ export default class EditModel {
 		this.commit = new Command();
 		this.cancel = new Command();
 		this.reset = new Command();
+		this.commitShortcuts = {
+			'$default': 'tab|enter',
```

Comment:
for the text tab is ok too, also tab should go to the next cell from edit mode does it work?